README.md

```diff
@@ -170,25 +170,34 @@ pip install -r /PATH/TO/THIS/DIRECTORY/requirements.txt
 For a general introduction to installing modules, see
 [here](https://docs.python.org/3/installing/index.html).
 
-*Optional dependencies*:
-* [pycuda](https://pypi.org/project/pycuda/),
-  required for the `tCWFstatMapVersion=pycuda`
-  option of the `TransientGridSearch` class.
-  (Note: `pip install pycuda` requires a working `nvcc` compiler in your path.)
-* [pytest](https://docs.pytest.org) for running the test suite locally
-  (`python -m pytest tests.py`)
-* Developers are also highly encouraged to use
-  the [flake8](https://flake8.pycqa.org/en/latest/) linter
-  and [black](https://black.readthedocs.io) style checker
-  locally,
-  as these checks are required to pass by the online integration pipeline.
-* Some optional plotting methods depend on the additional package
-  [chainconsumer](https://github.com/Samreay/ChainConsumer)
-  and some of the [example scripts](./examples) require this to run.
-  For `pip` users, this is most conveniently installed by
-```
-pip install chainconsumer
+### Optional dependencies
+
+PyFstat manages optional dependencies through setuptool's `extras_require`.
+
+Available sets of optional dependencies are:
+
+* [chainconsumer](https://github.com/Samreay/ChainConsumer): Required to run some optional 
+plotting methods and some of the [example scripts](./examples).
+* [pycuda](https://pypi.org/project/pycuda/): Required for the `tCWFstatMapVersion=pycuda`
+  option of the `TransientGridSearch` class. (Note: Installing `pycuda` requires a working 
+  `nvcc` compiler in your path.)
+* style: Includes the [flake8](https://flake8.pycqa.org/en/latest/) linter
```

Comment:
use ``style`` and similar highlighting for all the tags to make clear they're the actual named tags and not just part of the README?

Reply:
I though links couldn't be highlighted. I was [`wrong`](https://www.youtube.com/watch?v=ROGOCFpcj2Q) :wink: